ansible/roles/lazy/tasks/main.yml

```diff
@@ -4,7 +4,6 @@
   with_items:
       - squid
       - httpd
-      - python-pulp-streamer
```

Comment:
We don't need to install python-pulp-streamer via dnf when it's being installed with setuptools
https://github.com/pulp/pulp/blob/master/streamer/setup.py

This is also what was pulling in the unwanted pulp-server dependency before.